app/models/import/builder.rb

```diff
@@ -0,0 +1,13 @@
+class Import
+  class Builder
+    def initialize(gqueries, id:, scaling:, **)
+      @gqueries    = gqueries
+      @scenario_id = id
+      @scaling     = scaling
+    end
+
+    def build(response = nil)
```

Comment:
Allright, thanks for the explanation 👍 I rubocopped my way through the files and I think it's allright now.